.travis.yml

```diff
@@ -1,14 +1,18 @@
 os:
   - linux
+dist: bionic
```

Comment:
Why do you need this specific dist?

Reply:
I pinned it to use the latest distribution (18.04) because default is 16.04.